render.js

```diff
@@ -279,7 +280,7 @@ function html(results) {
 
                 res += (
                     '<div class="error_stack"' + (first ? '' : ' style="margin-top:2em;"') + '>' +
-                    escape_html(tr.error_stack || 'INTERNAL ERROR: no error stack') +
+                    escape_html(stripColors(tr.error_stack || 'INTERNAL ERROR: no error stack')) +
```

Comment:
@phihag I was unsure if I should do it here or if we should strip the colors when we assign `tr.error_stack`. What do you think? Happy to change it.

Reply:
Definitely the latter; we should have `tr.error_stack` unencumbered, and only apply coloring when outputting.

Reply:
Alright, updated the PR accordingly 🎉